src/logic/calculate.test.js

```diff
@@ -0,0 +1,16 @@
+import calculate from './calculate';
+
+describe('Sum 2 + 2', () => {
+  it('to be 4', () => {
+    let data = {
+      total: null,
+      next: null,
+      operation: null,
+    };
+    data = calculate(data, '2');
+    data = calculate(data, '+');
+    data = calculate(data, '2');
+    data = calculate(data, '=');
+    expect(Number(data.total)).toBe(4);
+  });
+});
```

Comment:
Nice work with the unit test for the `calculate.js` functionalities :+1: However, you should cover all the scenarios. The `if statements` in the `calculate` function are good indicators of the scenarios that should be tested. You can also check the comments in the file for more direction.